Speech/samples/V1/SpeechTranscribeAsyncWordTimeOffsetsGcs.php

```diff
@@ -0,0 +1,96 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * DO NOT EDIT! This is a generated sample ("LongRunningRequest",  "speech_transcribe_async_word_time_offsets_gcs")
+ */
+
+// sample-metadata
+//   title: Getting word timestamps (Cloud Storage) (LRO)
+//   description: Print start and end time of each word spoken in audio file from Cloud Storage
+
+//   usage: php samples/V1/SpeechTranscribeAsyncWordTimeOffsetsGcs.php [--storage_uri "gs://cloud-samples-data/speech/brooklyn_bridge.flac"]
+// [START speech_transcribe_async_word_time_offsets_gcs]
+require __DIR__.'/../../vendor/autoload.php';
+
+use Google\Cloud\Speech\V1\SpeechClient;
+use Google\Cloud\Speech\V1\RecognitionAudio;
+use Google\Cloud\Speech\V1\RecognitionConfig;
+
+/**
+ * Print start and end time of each word spoken in audio file from Cloud Storage.
+ *
+ * @param string $storageUri URI for audio file in Cloud Storage, e.g. gs://[BUCKET]/[FILE]
+ */
+function sampleLongRunningRecognize($storageUri)
+{
+    $speechClient = new SpeechClient();
+
+    // $storageUri = 'gs://cloud-samples-data/speech/brooklyn_bridge.flac';
+
+    // When enabled, the first result returned by the API will include a list
+    // of words and the start and end time offsets (timestamps) for those words.
+    $enableWordTimeOffsets = true;
+
+    // The language of the supplied audio
+    $languageCode = 'en-US';
+    $config = new RecognitionConfig();
+    $config->setEnableWordTimeOffsets($enableWordTimeOffsets);
+    $config->setLanguageCode($languageCode);
+    $audio = new RecognitionAudio();
+    $audio->setUri($storageUri);
+
+    try {
+        $operationResponse = $speechClient->longRunningRecognize($config, $audio);
+        $operationResponse->pollUntilComplete();
+        if ($operationResponse->operationSucceeded()) {
+            $response = $operationResponse->getResult();
+            // The first result includes start and end time word offsets
+            $result = $response->getResults()[0];
+            // First alternative is the most probable result
+            $alternative = $result->getAlternatives()[0];
+            printf('Transcript: %s'.PHP_EOL, $alternative->getTranscript());
+            // Print the start and end time of each word
+            foreach ($alternative->getWords() as $word) {
+                printf('Word: %s'.PHP_EOL, $word->getWord());
+                printf('Start time: %s seconds %s nanos'.PHP_EOL, $word->getStartTime()->getSeconds(), $word->getStartTime()->getNanos());
```

Comment:
Lines formatted like this break our limit of 120 characters, would it be possible to format this as a multi-line statement? :)

Reply:
It's very hard to do this in the generator :( Would it be possible to add a formatter rule here (or in artman)?

Reply:
I'll take a look and see if there's any tooling we can use :).